grammar.js

```diff
@@ -1616,7 +1622,7 @@ module.exports = grammar({
             choice(
               seq(
                 "\\",
-                choice(/[^xu]/, /uu?[0-9a-fA-F]{4}/, /x[0-9a-fA-F]{2}/),
+                choice(/[^xu]/, /[uU]+[0-9a-fA-F]{4}/, /x[0-9a-fA-F]{2}/),
```

Comment:
I have a comment noting this elsewhere, but Java weirdly allows any number of `u`s and `U`s at the start of an escape sequence. https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html